src/scripts/public/dom-loader.js

```diff
@@ -131,6 +141,75 @@ const domReady = () => {
             body.style.paddingTop = 0;
         }
 
+        /**
+         * Set up external icons for links.
+         */
+
+        if ('1' === window.site.externalLinkIcons) {
+            const links = document.querySelectorAll('#post-content a');
+
+            if (links) {
+                links.forEach((link) => {
+                    const href = link.getAttribute('href');
+
+                    // Check if the link is an anchor link or a relative link
+                    if (
+                        href.startsWith('#') ||
+                        href.startsWith('/') ||
+                        href.startsWith('./') ||
+                        href.startsWith('../')
```

Comment:
The error in my other comment was caused by pagination links in a query loop which start with "?" (`?query-1-page=2` for example). Might need to handle that case here too.

Reply:
Is a query string only URL valid? I've always been under the impression it would at least need to start with a path identifier, eg: /?query...

Reply:
> Is a query string only URL valid? I've always been under the impression it would at least need to start with a path identifier, eg: /?query...

Yeah I didn't know that was a valid value for href but that's how WP's Query Loops generate their pagination links.

Reply:
Added it just in case ;)